src/unittest/test_serialization.cpp

```diff
@@ -708,6 +711,51 @@ void TestSerialization::testFloatFormat()
 		UASSERT(test_single(i));
 }
 
+void TestSerialization::testDoubleFormat()
+{
+	std::ostringstream os(std::ios_base::binary);
+
+	for (int i = 0; i < 150; i++)
+	{
+		writeD1000(os, test_double_data[i]);
+		writeD(os, test_double_data[i]);
+		writeF32(os, test_double_data[i]);
+		writeF32(os, test_double_data[i]);
```

Comment:
why twice?

Reply:
That was a mistake. It was supposed to write using writeF32(ostream, double) and once using writeF32(ostream, f32) and verify that the two functions result in the same f32 being returned by readF32().

Reply:
Since writeF32 is no longer overloaded to take a double (see comment by @sfan5 below), I removed this subtest.